### PR TITLE
[SE-3221] Update README and provide screenshots of functionality.

### PR DESCRIPTION
**Pull request by Kelketek**
This pull request adds some more information about this repository, and its capabilities, to the README, adding some screenshots as well.

**Jira Ticket**: [OSPR-4969](https://openedx.atlassian.net/browse/OSPR-4969)

**Dependencies**: None

**Merge deadline**: Before its no longer accurate.

**Author notes and concerns**:

This is based on a discovery that was done by OpenCraft a couple of weeks ago. As this repository is quite actively developed, please make sure that the information contained therein is still accurate. If not, depending on time constraints I may have to update this and put it in our next sprint.

**Reviewers**
- [x] @giovannicimolin
- [ ] edX reviewer[s] TBD

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Kelketek! I've created [OSPR-4969](https://openedx.atlassian.net/browse/OSPR-4969) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

Please let us know once your PR is ready for our review and all tests are green.

**Review by giovannicimolin — APPROVED** [review]
:+1:

- [ ] ~~I tested this~~ NA
- [x] I read through the code
- [ ] ~~I checked for accessibility issues~~ NA
- [x] Includes documentation: improves documentation.

@Kelketek Good to go! Nice work! :)

**Comment by giovannicimolin** [comment]
@natabene This is ready for review. Mostly a minor documentation change better explaining what this repo is and who should use it. Similar to https://github.com/edx/frontend-app-publisher/pull/501

**Comment by natabene** [comment]
@giovannicimolin Thanks for letting me know. @nedbat Do you want to review or prefer that I reach out to the repo owners?

File: README.md
Comment (nedbat) [inline README.md:15]: We prefer "legacy" to describe the older implementation (throughout).

File: README.md
Comment (nedbat) [inline README.md:19]: It would be better to spell out "micro-frontend" where you can.

**Review by nedbat — DISMISSED** [review]

File: README.md
Comment (Kelketek) [reply README.md]: Changed.

File: README.md
Comment (Kelketek) [reply README.md]: Changed.

**Comment by Kelketek** [comment]
@nedbat Made the changes you requested. Thank you.

**Comment by Kelketek** [comment]
@nedbat Would we be able to get this looked at this coming week or the next?

**Comment by mattcarter** [comment]
@nedbat, if you're satisfied with the revised changes, we can merge.

**Comment by nedbat** [comment]
@mattcarter I good with it if you are! Merge it.

**Comment by natabene** [comment]
@mattcarter Should we merge this, since everyone is in favor?

**Review by mattcarter — APPROVED** [review]
approved

**Comment by mattcarter** [comment]
@nedbat - I'm getting blocked on merging. I'm dismissing your open change request. We look good from the comment stream.

**Comment by edx-webhook** [comment]
@Kelketek 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/frontend-app-gradebook&entry.867055334=https://github.com/edx/frontend-app-gradebook/pull/143&entry.1484655318=https://github.com/Kelketek&entry.752974735=2020-09-14+17:57&entry.1917517419=2020-11-23+17:48&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
